docs/sources/datasources/postgres/configure/_index.md

```diff
@@ -89,8 +89,17 @@ Following is a list of PostgreSQL configuration options:
 
 **Connection section:**
 
-- **Host URL** - The IP address/hostname and optional port of your PostgreSQL instance.
-- **Database name** - The name of your PostgreSQL database.
+- **Host URL** - The IP address/hostname and optional port of your PostgreSQL instance. You can specify the host in several formats:
+  - Simple hostname: `localhost`
+  - Host with port: `localhost:5432`
+  - IPv4 address: `192.168.1.1:5432`
+  - IPv6 address: `[2001:db8::1]:5432`
+  - Full PostgreSQL URL: `postgresql://user:password@host:5432/dbname`
```

Comment:
Do we want to add this as a last option and mention that this is not recommended as password is in this cased not saved as secure field?